Webspage 3.0/index.htm

```diff
@@ -1,15 +1,21 @@
 <!DOCTYPE html>
 <html>
 <head>
-<link rel="stylesheet" type="text/css" href="mystyles.css">
+<link rel="stylesheet" type="text/css" href="styles/main.css">
 </head>
 
 <body>
 
-<img src="img/gauge_menu.png" style="float:right; width: 275px; height: 500px;">
-<h1>Mike Elrod</h1>
-<h2>Portfolio</h2>
 
+	<h1>MIKE ELROD</h1>

```

Comment:
Excellent. You will want an H1 on every page.  It tells search engines what the page is about and helps SEO.